greedybear/celery.py

```diff
@@ -86,4 +86,10 @@ def setup_loggers(*args, **kwargs):
         "schedule": crontab(minute=33),
         "options": {"queue": "default"},
     },
+    # once a day
```

Comment:
ok but the DB_BACKUP_FREQUENCY varialbe is never used. It will always be executed once a day...

Reply:
Sir, I've added the backup frequency to the celery.py. It should work now. Please review.